src/index.js

```diff
@@ -1,14 +1,26 @@
 import 'dotenv/config';
 import cors from 'cors';
 import express from 'express';
+import query from './db/index';
+import { readQuestions } from './db/transformCSV';
+
+// Import routes
+import indexRouter from './routes/index';
+import qaRoutes from './routes/qa';
 
 const app = express();
 
 app.use(cors());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
```

Comment:
What does this line do?

Reply:
If you're referring the the Import routes, it's just a way of breaking my code into smaller/separate files. I will review in the future if it's necessary as I'm not sure how big it will get yet.